pkg/lib/cockpit-components-select.jsx

```diff
@@ -16,215 +16,206 @@
  * You should have received a copy of the GNU Lesser General Public License
  * along with Cockpit; If not, see <http://www.gnu.org/licenses/>.
  */
+"use strict";
 
-(function() {
-    "use strict";
-
-    const cockpit = require("cockpit");
-    const React = require("react");
-
-    require("page.css");
-
-    const _ = cockpit.gettext;
-
-    const textForUndefined = _("undefined");
-
-    /* React pattern component for a dropdown/select control
-     * Entries should be child components of type SelectEntry (html <a>)
-     *
-     * User of this component should listen onChange and set selected prop of it
-     *
-     * Expected properties:
-     *  - selected (optional) explicit data to select, default: first entry
-     *  - onChange (optional) callback (parameter data) when the selection has changed
-     *  - id (optional) html id of the top level node
-     *  - enabled (optional) whether the component is enabled or not; defaults to true
-     *  - extraClass (optional) CSS class name(s) to be added to the main <div> of the component
-     */
-    class StatelessSelect extends React.Component {
-        constructor() {
-            super();
-            this.clickHandler = this.clickHandler.bind(this);
-
-            this.state = {
-                open: false,
-                documentClickHandler: undefined,
-            };
-        }
+import ReactDOM from 'react-dom';
+import React from 'react';
+import PropTypes from 'prop-types';
+import cockpit from "cockpit";
 
-        componentDidMount() {
-            const handler = this.handleDocumentClick.bind(this, React.findDOMNode(this));
-            this.setState({ documentClickHandler: handler });
-            document.addEventListener('click', handler, false);
-        }
+require("./page.css");
```

Comment:
This can be `import "page.css"` as well.  And we should probably keep the old path "page.css" instead of "./page.css".